### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by manvc**
Hey I noticed you have used unicode library too. Thought it should be in the requirements.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 pandas
 geopandas
 altair
-requests_html
\ No newline at end of file
+requests_html
+unicode
```
Comment (armsp) [inline requirements.txt]: So, there is no library called `unicode`. Did you mean `unicodedata` ?

**Review by armsp — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (manvc) [reply requirements.txt]: Sorry my bad. I meant unicodedata. I have corrected the file.

File: requirements.txt [hunk 2]
```diff
@@ -2,4 +2,4 @@ pandas
 geopandas
 altair
 requests_html
-unicode
+unicodedata
```
Comment (armsp) [inline requirements.txt:5]: So, `unicodedata` is part of the [standard library](https://docs.python.org/3/library/). So I don't think it's needed in the _requirements.txt_ file.

**Comment by armsp** [comment]
Closing because no changes are required

**Pull request closed without merging**